README.md

```diff
@@ -1,457 +1,76 @@
+A typical resource-server requires a frontend and a backend application, so you will need to start each process:
 
-```javascript
-// login-redirect.controller.js
-
-class LoginRedirectController {
-   constructor(config) {
-    this.config = config;
-  }
-   $onInit() {
-    this.authClient = new OktaAuth({
-      url: this.config.oktaUrl,
-      issuer: this.config.issuer,
-      clientId: this.config.clientId,
-      redirectUri: this.config.redirectUri,
-      scopes: ['openid', 'email', 'profile'],
-    });
-  }
- 
-  login() {
-    this.authClient.token.getWithRedirect({ responseType: 'code' });
-  }
-}
+backend:
+```bash
+cd resource-server
+mvn -Dokta.oauth2.issuer=https://dev-123456.oktapreview.com/oauth2/default
 ```
 
-There are a number of different ways to construct the login redirect URL.
-
-1. Build the URL manually
-2. Use an OpenID Connect / OAuth 2.0 middleware library
-3. Use [AuthJS](http://developer.okta.com/code/javascript/okta_auth_sdk)
-
-In this sample, we use AuthJS to create the URL and perform the redirect. An `OktaAuth` object is instantiated with the configuration in `.samples.config.json`. When the `login()` function is called from the view, it calls the [`/authorize`](http://developer.okta.com/docs/api/resources/oauth2.html#authentication-request) endpoint to start the [Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1).
-
-You can read more about the `OktaAuth` configuration options here: [OpenID Connect with Okta AuthJS SDK](http://developer.okta.com/code/javascript/okta_auth_sdk#social-authentication-and-openid-connect).
-
-**Important:** When the authorization code is exchanged for an `access_token` and/or `id_token`, the tokens **must** be [validated](#validation). We'll cover that in a bit.
+front-end:
 
-### Custom Login Form
-To render the [Okta Sign-In Widget](http://developer.okta.com/code/javascript/okta_sign-in_widget), include a container element on the page for the widget to attach to:
+This project is just serving a static `index.html` so you WILL need to update the configuration in that file, see line 53:
 
-```html
-<!-- overview.mustache -->
-<div id="sign-in-container"></div>
-```
-
-Then, initialize the widget with the [OIDC configuration](https://github.com/okta/okta-signin-widget#openid-connect) options:
-``` javascript
-// login-custom.controller.js
-class LoginCustomController {
-  constructor(config) {
-    this.config = config;
-  }
- 
-  $onInit() {
-    const signIn = new SignIn({
-      baseUrl: this.config.oktaUrl,
-      clientId: this.config.clientId,
-      redirectUri: this.config.redirectUri,
-      authParams: {
-        issuer: this.config.issuer,
-        responseType: 'code',
-        scopes: ['openid', 'email', 'profile'],
-      },
-    });
-    signIn.renderEl({ el: '#sign-in-container' }, () => {});
+```javascript
+const data = {
+  baseUrl: 'https://dev-123456.oktapreview.com',
+  clientId: 'YourClientId',
+  redirectUri: 'http://localhost:8080',
```

Comment:
redirectUri for SPA is `http://localhost:8080/implicit/callback` by default
Also, should we mention that the user needs to have an SPA created in okta to test resource server flow?

Reply:
`implicit/callback` doesn't work in this case as it is only a static file.
This front end piece will be replaced with one of the front end libs once those are in place